### PR TITLE
Keyboard focus on owner list on package detail page

### PR DESCRIPTION
**Pull request by lyndaidaii**
Summary of the changes (in less than 80 characters):

Before: 
image

After:
image

Addresses: https://github.com/NuGet/Engineering/issues/4827

### CODE REVIEW

**Comment by joelverhagen** [comment]
Can you test this with a very long username? And a very long username containing a character like a hyphen?

**Review by joelverhagen — DISMISSED** [review]
LGTM, I just wanted to make sure long usernames were tested.

**Review by theot12 — APPROVED** [review]

**Comment by lyndaidaii** [comment]
> LGTM, I just wanted to make sure long usernames were tested.

tested

**Comment by lyndaidaii** [comment]
> LGTM, I just wanted to make sure long usernames were tested.

original doesn't work for long username.  Put new changes. 

MicrosoftTeams-image (5)

**Comment by joelverhagen** [comment]
I don't think we should widen the panel. Hiding the overflow with ellipsis (this is previous behavior) is right.

File: src/Bootstrap/less/theme/page-display-package.less [hunk 1]
```diff
@@ -245,14 +245,22 @@
     li {
       margin-bottom: 14px;
       display: block;
-      overflow: hidden;
+      overflow:hidden;
```
Comment (joelverhagen) [inline src/Bootstrap/less/theme/page-display-package.less:248]:
```suggestion
      overflow: hidden;
```

**Review by joelverhagen — APPROVED** [review]

**Pull request merged**